data/jibauni/rebel jibauni 0 consultations.txt

```diff
@@ -0,0 +1,362 @@
+# Copyright (c) 2024 by Loymdayddaud
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "First Contact: Rebel Jibauni"
+	name "Travel to <planet>"
+	description `Travel to <destination> to meet up with the leaders of this "Rebel Jibauni" faction.`
+	landing
+	source
+		government "Jibauni (Rebel)"
+	destination Andwerfen
+	to offer
+		not "Jibauni: Kanlanmo 0: active"
+		not "Jibauni: Kanlanmo 0: done"
+		not "Jibauni: Kanlanmo (0 declined): active"
+		not "Jibauni: Kanlanmo (0 declined): done"
+	on offer
+		conversation
+			branch "flagship andwerfen"
+				has "flagship planet: Andwerfen"
+			branch "met and declined the jibauni"
+				has "jibauni: joined the rebels"
+				not "flagship planet: Andwerfen"
+			branch "met and declined the jibauni and on andwerfen"
+				has "jibauni: joined the rebels"
+				not "flagship planet: Andwerfen"
+			label nothing
+			`As you land on <origin>, a strange creature comes up to you. It is in the shape of the fabled Tyrannosaurus Rex from Earth, but it is covered in what is seemingly metal armor.`
+			`	Dangerous, yes, but it might also be wise to go out and attempt to communicate with the mysterious dinosaur.`
+			choice
+				`	(Go try to speak with the alien.)`
+					goto speak
+				`	(Leave. The danger isn't worth it.)`
+					goto flee
+				`	(The unknown is dangerous. Shoot the thing.)`
+					goto shot
+			label speak
+			`	Before you can do anything, you hear strange words, something like "Je. Vqekuves?" Suddenly the words are converted to a language you can barely understand. "Go to <planet>." Apparently that is the only thing they have to say, and the huge crowd disperses after watching you make your preparations for taking off... and starting the launch sequence.`
```

Comment:
Take everything I say with a grain of salt.  Just see it as possible ideas for changes. 

```suggestion
			`	Before you can do anything, incomprehensible sounds erupt from the alien's mouth, something like "Je. Vqekuves?" Suddenly the words are snatched out of the air and twisted into a words simple understandable words. "Go to <planet>." Apparently that is the only thing they have to say, and the huge crowd disperses after watching you make your preparations for taking off... and starting the launch sequence.`
```